MANUAL.txt

```diff
@@ -3176,6 +3176,17 @@ In the `context` output format this enables the use of [Natural Tables
 Natural tables allow more fine-grained global customization but come
 at a performance penalty compared to extreme tables.
 
+#### Extension: `native_figures` ####
+
+Use pandoc's native `Figure` element for content inside `<figure>` tags, in the
+case of HTML, or `figure` environments, in case of LaTeX. This, in turn, allows
+some writers to produce more accurate representations of figures. It also
+allows the use of the `Figure` element in filters, for custom figure output.
+
+This extension can be enabled/disabled for the following formats:
+
+input formats
+: `latex` `html`
```

Comment:
Question:  do we really need to gate this under an extension?  What is the motivation for that?
An alternative would be to always use Figure elements to represent figures.

Reply:
The motivation to gate this under an extension is to keep backward compatibility. Some users may still rely on the original representation in their lua-filters.

> An alternative would be to always use Figure elements to represent figures.

Or this extension could be turned on by default allowing users to fallback to the original behaviour.

Reply:
I'll comment in the main comments.